src/components/wordLogo.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+
+const WordLogo = (props: any) => (
```

Comment:
Can you specify a proper `interface` for these props and use that instead of `any`?